mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -957,7 +974,7 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
         };
 
         let prepared_params = TakerSwapPreparedParams {
-            dex_fee: Default::default(),
+            dex_fee: Default::default(), // TODO: is it correct to use 0 here? It's for calculating the needed balance. What if the actual dex_fee will be not 0?
```

Comment:
yep, could you explain why you dont calc dex fee here like in legacy
![image](https://github.com/user-attachments/assets/3a281909-3fb0-4e34-9244-cfdbc529bdb5)